pkg/util/tracing/tracer.go

```diff
@@ -179,7 +179,7 @@ var EnableActiveSpansRegistry = settings.RegisterBoolSetting(
 	settings.ApplicationLevel,
 	"trace.span_registry.enabled",
 	"if set, ongoing traces can be seen at https://<ui>/#/debug/tracez",
-	envutil.EnvOrDefaultBool("COCKROACH_REAL_SPANS", true),
+	envutil.EnvOrDefaultBool("COCKROACH_REAL_SPANS", false),
```

Comment:
Agree this is the right call. I haven't observed much usage of `tracez` in the last year.